Content.Server/Medical/SuitSensors/SuitSensorSystem.cs

```diff
@@ -429,6 +439,7 @@ public NetworkPayload SuitSensorToPacket(SuitSensorStatus status)
 
         // try get total damage and cords (optionals)
         payload.TryGetValue(SuitSensorConstants.NET_TOTAL_DAMAGE, out int? totalDamage);
+        payload.TryGetValue(SuitSensorConstants.NET_TOTAL_DAMAGE_THRESHOLD, out int? totalDamageThreshold);
```

Comment:
This doesn't look like it's actually used after being read. Shouldn't it be passed into the SuitSensorStatus below?

Reply:
Wow, don't know how I missed that, my bad. Thanks for catching it.